README.md

```diff
@@ -80,24 +79,181 @@ After you input your application credentials, you can click the setup button, wh
 
 **Important**: Make sure your Resources/MobiledgeXSettings.asset file has the correct information for your application. 
 
-Once that setup has been completed, you can very easily call all the necessary API requests to connect to a cloudlet with your application deployed. Here is some example code using the MobiledgeXIntegration class that comes with the package : 
+Once that setup has been completed, you can very easily call all the necessary API requests to connect to a cloudlet with your application deployed. Here is some example code using the MobiledgeXIntegration class that comes with the package 
+
+
+**Getting Edge Connection Url**
+
+
 
 ```csharp
 using MobiledgeX;
+using DistributedMatchEngine;
+using UnityEngine;
+using System.Collections;
+
+[RequireComponent(typeof(MobiledgeX.LocationService))]
+public class NetworkManager : MonoBehaviour
+{ 
+    IEnumerator Start()
+    {
+        yield return StartCoroutine(MobiledgeX.LocationService.EnsureLocation()); // Location is needed to connect you to the closet edge
+        GetEdgeConnection();
+    }
+     async void GetEdgeConnection()
+    {
+        MobiledgeXIntegration mxi = new MobiledgeXIntegration();
+        try
+        {
+            await mxi.RegisterAndFindCloudlet();
+        }
+        catch(DmeDnsException)
+        {
+            mxi.UseWifiOnly(true); // if you carrier is not supported yet, WifiOnly will connect you to the closet MobiledgeX Public Cloud
+            await mxi.RegisterAndFindCloudlet();
+        }
+
+        mxi.GetAppPort(LProto.L_PROTO_HTTP); // Get the port of the desired protocol
+        string url = mxi.GetUrl("http"); // Get the url of the desired protocol
+    }
+    
+}
+```
+
+
+MobiledgeX requires Location and Mobile phone carrier information (specifically MccMnc code)
+
+
+If you carrier is not supported yet by MobiledgeX it will throw a DmeDnsException and it will connect you to the closet MobiledgeX Public Cloud 
```

Comment:
This statement isnt exactly true. For the DmeDnsException, refer to what I wrote in the FAQ

Reply:
I don't understand excatly
Do you mean replace 
If you carrier is not supported yet by MobiledgeX it will throw a DmeDnsException and it will connect you to the closet MobiledgeX Public Cloud 
with
Since MobiledgeX is currently in private beta, we have not created DME entries for all mcc-mnc values. Please reach out to us on Slack or e-mail Support so that we can create a DME with your mcc-mnc value.

Or you dont like the try catch in the example code?

Reply:
oh I see what you are trying to refer to. I think it i was little unclear that this was a reference to the try catch. for context, i think you can add something like "the reason for the try catch DmeDnsException is because if your carrier is not supported...."

Reply:
the second part is inaccurate though since we dont connect you the public cloud